keanu-project/src/main/java/io/improbable/keanu/vertices/dbl/probabilistic/GaussianVertex.java

```diff
@@ -101,6 +103,26 @@ public double logProb(DoubleTensor value) {
         return logPdfs.sum();
     }
 
+    @Override
+    public LogProbGraph logProbGraph() {
+        final LogProbGraph.DoublePlaceHolderVertex xInput = new LogProbGraph.DoublePlaceHolderVertex(this.getShape());
```

Comment:
I made it a nested class because it's easy to confuse it with a proxy vertex without the context of this specific use case. It's easy to pull it out later when we need it for other things.

Reply:
ok